analysis.json

```diff
@@ -1,5 +1,8 @@
 {
   "dependencies": {
+    "vaadinVersions": [ "7.7.40", "8.20.3"],
```

Comment:
@heruan do we need here 4 versions for 8 and 4 versions for 7 which is what we agreed to maintain?
How is this array used?
Is the case that a user running appsec in 8.8.0 will not work since that version is not in the array?